op-challenger/challenger.go

```diff
@@ -1,12 +1,50 @@
 package op_challenger
 
 import (
+	"fmt"
+	"time"
+
+	"github.com/ethereum-optimism/optimism/op-bindings/bindings"
 	"github.com/ethereum-optimism/optimism/op-challenger/config"
+	"github.com/ethereum-optimism/optimism/op-challenger/fault"
+	"github.com/ethereum-optimism/optimism/op-service/txmgr"
+	"github.com/ethereum-optimism/optimism/op-service/txmgr/metrics"
+	"github.com/ethereum/go-ethereum/ethclient"
 	"github.com/ethereum/go-ethereum/log"
 )
 
 // Main is the programmatic entry-point for running op-challenger
 func Main(logger log.Logger, cfg *config.Config) error {
+	client, err := ethclient.Dial(cfg.L1EthRpc)
+	if err != nil {
+		return fmt.Errorf("failed to dial L1: %w", err)
+	}
```

Comment:
Can we log an issue to make the challenger handle starting even if it's L1 isn't available. Don't need it straight away but worth doing fairly early before it becomes hard to handle properly. k8s doesn't do ordering of pods well and it's really annoying to have programs exit because a dependency isn't fully started yet.

Reply:
Really just needs to use the pre-built dial with retry